src/vconsole/vconsole-setup.c

```diff
@@ -282,6 +287,7 @@ int main(int argc, char **argv) {
         utf8 = is_locale_utf8();
 
         r = parse_env_file("/etc/vconsole.conf", NEWLINE,
+                           "VC_CNT", &vc_cnt_s,
```

Comment:
Hmpf, I really don't like exposing such abbreviated variables. We are not trying to save individual bytes here, and adding the OU in there, turning this into VC_COUNT come at the price of JUST TWO BYTES and the benefit of being a lot more self-explanatory. I'd clal that a bargain!

logind.conf knows two settings NAutoVTs= and ReserveVT=. In order to be more in line with thate we should probably call this field something with "VT" rather than "VC". Maybe N_ALLOCATE_VTS= or so? or N_APPLY_VTS= or so?